Account/Sources/Account/WebFeed.swift

```diff
@@ -220,6 +225,16 @@ public final class WebFeed: Feed, Renamable, Hashable {
 			postUnreadCountDidChangeNotification()
 		}
 	}
+    
+    // MARK: - Feed Provider
+    public var isFeedProvider: Bool {
+        get {            
+            if FeedProviderManager.shared.best(for: URLComponents(url: URL(string: url)!, resolvingAgainstBaseURL: false)!) == nil {
```

Comment:
I don't think we should be force unwrapping here. A bad URL, however unlikely, would cause crashes. Let's return false if it is a bad URL.

Reply:
Fixed in 0fa30a94.